tests/ttnn/integration_tests/stable_diffusion/test_cross_attn_up_block_2d.py

```diff
@@ -214,7 +214,7 @@ def test_cross_attn_up_block_2d_512x512(
     out_channels,
 ):
     # TODO
-    # pytest.skip()
+    pytest.skip()
```

Comment:
Hey do you mean writing a reason string with an issue number so we can track the work, instead of just an empty skip?

Reply:
For all of them

Reply:
These tests were failing when I had started with the SD uplift, and were skipped. I had enabled them while I was working on SD, and had forgotten to disabled them. 
I will ask Aleks if these are to be fixed.

Reply:
Is there an update?